test/e2e/clusterlogging_test.go

```diff
@@ -236,3 +242,49 @@ func clusterLoggingUpgradeTest(t *testing.T, f *framework.Framework, ctx *framew
 	t.Log("Completed ClusterLogging upgrade test")
 	return nil
 }
+
+func changeLoggingCollectorTest(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, collector string) error {
+	t.Log("Starting change logging collector test...")
+	t.Logf("Logging Collection type is: %s", collector)
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		return fmt.Errorf("Could not get namespace: %v", err)
+	}
+	newcollector := ""
+	exampleClusterLogging := &logging.ClusterLogging{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ClusterLogging",
+			APIVersion: logging.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "instance",
+			Namespace: namespace,
+		},
+	}
+
+	exampleName := types.NamespacedName{Name: exampleClusterLogging.GetName(), Namespace: namespace}
+	if err = f.Client.Get(goctx.TODO(), exampleName, exampleClusterLogging); err != nil {
+		return fmt.Errorf("failed to get exampleClusterLogging: %v", err)
+	}
+
+	if exampleClusterLogging.Spec.Collection.Logs.Type == "fluentd" {
```

Comment:
I think there are defined constants for these types

Reply:
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/apis/logging/v1/types.go#L220-L221

Reply:
Thanks, I'll change it to use const vars.